app/actions/Session.js

```diff
@@ -0,0 +1,284 @@
+import { ipcRenderer } from 'electron';
+import settings from 'electron-settings';
+import { v4 as UUID } from 'uuid';
+import { push } from 'react-router-redux';
+
+export const NEW_SESSION_REQUESTED = 'NEW_SESSION_REQUESTED';
+export const NEW_SESSION_BEGAN = 'NEW_SESSION_BEGAN';
+export const NEW_SESSION_DONE = 'NEW_SESSION_DONE';
+export const CHANGE_CAPABILITY = 'CHANGE_CAPABILITY'
+export const SAVE_SESSION_REQUESTED = 'SAVE_SESSION_REQUESTED';
+export const SAVE_SESSION_DONE = 'SAVE_SESSION_DONE';
+export const GET_SAVED_SESSIONS_REQUESTED = 'GET_SAVED_SESSIONS_REQUESTED';
+export const GET_SAVED_SESSIONS_DONE = 'GET_SAVED_SESSIONS_DONE';
+export const SET_CAPABILITY_PARAM = 'SET_CAPABILITY_PARAM';
+export const ADD_CAPABILITY = 'ADD_CAPABILITY';
+export const REMOVE_CAPABILITY = 'REMOVE_CAPABILITY';
+export const SWITCHED_TABS = 'SWITCHED_TABS';
+export const SET_CAPS = 'SET_CAPS';
+export const SAVE_AS_MODAL_REQUESTED = 'SAVE_AS_MODAL_REQUESTED';
+export const HIDE_SAVE_AS_MODAL_REQUESTED = 'HIDE_SAVE_AS_MODAL_REQUESTED';
+export const SET_SAVE_AS_TEXT = 'SET_SAVE_AS_TEXT';
+export const DELETE_SAVED_SESSION_REQUESTED = 'DELETE_SAVED_SESSION_REQUESTED';
+export const DELETE_SAVED_SESSION_DONE = 'DELETE_SAVED_SESSION_DONE';
+export const CHANGE_SERVER_TYPE = 'CHANGE_SERVER_TYPE';
+export const SET_SERVER_PARAM = 'SET_SERVER_PARAM';
+export const SET_SERVER = 'SET_SERVER';
+export const SESSION_LOADING = 'SESSION_LOADING';
+export const SESSION_LOADING_DONE = 'SESSION_LOADING_DONE';
+
+const SAVED_SESSIONS = 'SAVED_SESSIONS';
+const SESSION_SERVER_PARAMS = 'SESSION_SERVER_PARAMS';
+const SESSION_SERVER_TYPE = 'SESSION_SERVER_TYPE';
+
+export const ServerTypes = {
+  local: 'local',
+  remote: 'remote',
+  sauce: 'sauce',
+};
+
+function getCapsObject (caps) {
```

Comment:
I switched it to

```javascript
Object.assign({}, ...(caps.map((cap) => { 
    return {[cap.name]: cap.value}; 
  })));
```

It confuses the opening and clothing curly braces as being the curly braces for a function and not a returned object